src/mixins/nft.js

```diff
@@ -477,10 +479,14 @@ export default {
     },
     async fetchNFTPrices(classId) {
       try {
-        const { fiatPrice } = await this.$axios.$get(
+        const { fiatPrice, isListing, listingInfo } = await this.$axios.$get(
           getStripeFiatPrice({ classId })
         );
         this.nftPriceInUSD = fiatPrice;
+        if (isListing) {
+          this.listingNftId = listingInfo.nftId;
+          this.listingNftSeller = listingInfo.seller;
+        }
```

Comment:
Can we make use of `nftIsUseListingPrice` and `listingInfo` instead of making duplicate new fields? Or at least we should refactor current useListing logic when buying with LIKE into the new fields